spec/helpers/application_helper_spec.rb

```diff
@@ -255,4 +255,28 @@
       expect(optimized_helper).to eq(cloudinary_image_tag)
     end
   end
+
+  describe "#cache_multi" do
+    it "reads many keys in a single call" do
+      cache = instance_double("ActiveSupport::Cache::Store")
+      helper.controller.perform_caching = true # ensure we invoke caching here
+
+      allow(cache)
+        .to receive(:fetch_multi)
+        # Must account for the options hash for things like :expires_in
+        .with(:foo, :bar, :baz, {})
```

Comment:
**nitpick (non-blocking):** Not a hard and fast rule but we tend to avoid the `foo`s and `bar`s and use names that reflect context a bit better.